drv/transceivers-server/src/udp.rs

```diff
@@ -687,6 +748,68 @@ impl ServerImpl {
         )
     }
 
+    /// This function reads a `ModuleResultNoFailure` and populates error
+    /// information at the end of the trailing data buffer, taking `self.data`
+    /// into account. This means it should be called as the last operation
+    /// before sending the response. For results where a `ModuleResult` is
+    /// returned, use `handle_errors_and_failures_and_disabled` instead.
+    fn handle_errors_and_disabled(
+        &mut self,
+        modules: ModuleId,
+        result: ModuleResultNoFailure,
+        out: &mut [u8],
+    ) -> (usize, ModuleId) {
+        let mut error_idx: usize = 0;
+        // any modules at index 32->63 are not currently supported.
+        let invalid_modules = ModuleId(0xffffffff00000000);
+        let requested_invalid_modules = ModuleId(modules.0 & invalid_modules.0);
```

Comment:
The `ModuleId` directly implements the bitwise ops, now, if you want to clean this up a bit.

Reply:
Nice, fixed consistently in 6e449bb0d32917051ae82edbc882ed7de189b0a7